cockatrice/src/userlist.cpp

```diff
@@ -276,7 +276,7 @@ void UserListTWI::setUserInfo(const ServerInfo_User &_userInfo)
     userInfo = _userInfo;
 
     setData(0, Qt::UserRole, userInfo.user_level());
-    setIcon(0, QIcon(UserLevelPixmapGenerator::generatePixmap(12, UserLevelFlags(userInfo.user_level()), false)));
+    setIcon(0, QIcon(UserLevelPixmapGenerator::generatePixmap(12, UserLevelFlags(userInfo.user_level()), false, QString::fromStdString(userInfo.privlevel()))));
```

Comment:
@ZeldaZach Even with this change I see the following when the user is offline:
![image](https://cloud.githubusercontent.com/assets/1361287/21012556/a48a5bce-bd24-11e6-9749-1a9652eb7c18.png)